source/common/http/conn_manager_impl.h

```diff
@@ -218,10 +217,9 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     }
 
     // ScopeTrackedObject
-    ExecutionContext* executionContext() const override {
-      return getConnectionExecutionContext(connection_manager_.read_callbacks_->connection());
+    OptRef<const StreamInfo::StreamInfo> trackedStream() const override {
+      return filter_manager_.trackedStream();
```

Comment:
Does `filter_manager_.trackedStream()` always return the connection level streamInfo?

(Sorry for the dump question, I'm not familiar with this code)

Reply:
It will return the request level stream info. But the filter state is tree level thing. It will eventually search to the connection level filter state.

Reply:
Got it, thanks.